app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -72,7 +69,6 @@ def update
     api :POST, '/organizations', N_('Create organization')
     param :name, String, :desc => N_("name"), :required => true
     param :label, String, :desc => N_("unique label")
-    param :description, String, :desc => N_("description")
```

Comment:
I guess it happens when you try this PR in combination with https://github.com/theforeman/foreman/pull/2037, as there is the description then defined twice: once in the organizations_controller and once in the foreman taxonomies_controller? Could that be the case?